framework/test_app/runners/k8s/k8s_base_runner.py

```diff
@@ -998,9 +998,12 @@ def _wait_pod_started(self, name, **kwargs) -> k8s.V1Pod:
         logger.info("Waiting for pod %s to start", name)
         self.k8s_namespace.wait_for_pod_started(name, **kwargs)
         pod = self.k8s_namespace.get_pod(name)
-        logger.info(
-            "Pod %s ready, IP: %s", pod.metadata.name, pod.status.pod_ip
-        )
+
+        if hasattr(pod.status, "pod_ip_s"):  # if running with dualstack support
+            pod_ips = pod.status.pod_ip_s
+        else:
+            pod_ips = pod.status.pod_ip
+        logger.info("Pod %s ready, IP: %s", pod.metadata.name, pod_ips)
```

Comment:
I see that we're not saving the ips anywhere, just logging. @larry-safran - are we going to need actual IPs anywhere? If yes, we should capture them in `_xds_test_client_for_pod`/`_xds_test_server_for_pod`, and save them to the new `ips` field in `XdsTestClient`/`XdsTestServer`

Reply:
Do you mean replacing existing ip field by ips and populate from kubernetes based on dualstack or not? Or you are suggesting to maintain separate field for ips and populate only in case of dualstack?

Reply:
We didn't need it before and it can always be retrieved from the `pod`.

Reply:
Thanks